src/Engines/engine.ts

```diff
@@ -290,6 +290,17 @@ export class Engine extends ThinEngine {
         return EngineStore.LastCreatedScene;
     }
 
+    /**
+     * Engine abstraction for createImageBitmap
+     */
+    public createImageBitmap(...args: any[]): Promise<ImageBitmap> //image: ImageBitmapSource, options?: ImageBitmapOptions
+    {
+        if (args.length <= 2) {
+            return createImageBitmap(args[0], args[1]);
+        }
+        return createImageBitmap(args[0], args[1], args[2], args[3], args[4], args[5]);

```

Comment:
I think you can probably use `createImageBitmap.apply` here?

Reply:
Exactly what I was looking for!